rust/config/mainnet_config.json

```diff
@@ -1,6 +1,7 @@
 {
   "chains": {
     "ancient8": {
+      "batchContractAddress": "0x4C97D35c668EE5194a13c8DE8Afc18cce40C9F28",
```

Comment:
why not call this multicall3?

Reply:
I wanna keep it generic to all execution envs since we may soon add support for cosmwasm

Reply:
gotcha
FYI cosmwasm has native batching